docs/sites.yml

```diff
@@ -5383,3 +5383,87 @@
   built_by: Nikodem Deja
   built_by_url: https://nikodemdeja.pl
   featured: false
+- title: manuvel.be
+  url: https://www.manuvel.be
+  main_url: https://www.manuvel.be
+  description: >
+    Cycling themed café coming this april in Sint Niklaas, Belgium. Has funky css-grid trickery!
+  categories:
+    - Café
+    - Instagram
+    - CSS Grid
+  built_by: Web Hart
+  built_by_url: https://www.web-hart.com
+  featured: false
+- title: WEB hart
+  url: https://www.web-hart.com
+  main_url: https://www.web-hart.com
+  description: >
+    Hi, I'm Rien (pronounced Reen) from Belgium but based in Girona, Spain. I'm an autodidact, committed to learning until the end of time.
+  categories:
+    - Portfolio
+    - Design
+    - Web Development
+    - Freelance
+  built_by: Web Hart
+  built_by_url: https://www.web-hart.com
+  featured: false
+- title: nicdougall.com
+  url: https://nicdougall.netlify.com/
+  main_url: https://nicdougall.netlify.com/
+  description: >
+    Athlete website with Netlify CMS for blog content.
+  categories:
+    - Athlete
+    - Blog
+    - Netlify CMS
+  built_by: Web Hart
+  built_by_url: https://www.web-hart.com
+  featured: false
+- title: het Groeiatelier
+  url: https://www.hetgroeiatelier.be/
+  main_url: https://www.hetgroeiatelier.be/
+  description: >
+    Workspace for talent development and logopedics. onepage site with basic info and small calendar CMS.
+  categories:
+    - logopedics
```

Comment:
This category seems very specific and likely to be one-off - ideally we could use something more generic that would apply to more sites to avoid categories that would hold just single entries